infra/net/handlers.go

```diff
@@ -0,0 +1,37 @@
+package net
+
+import (
+	"bootcamp/adapters"
+	"bootcamp/helper"
+	"fmt"
+	"net/http"
+)
+
+func NewGetCharacterHandler(getCharacterController adapters.GetCharacterController) func(writer http.ResponseWriter, request *http.Request) {
+	return func(writer http.ResponseWriter, request *http.Request) {
```

Comment:
Curiously enough, almost all of the handlers I've seen declare their writer and request params as `w` and `r` only. I'm not sure if this is by convention, but Go tends to have really short names for common identifiers like these.

Reply:
I prefer to use complete variable names, but iif it is somethings regular on go, I can adopt it.